index.client.js

```diff
@@ -56,17 +57,27 @@ ReactI13nGoogleAnalytics.prototype.getPlugin = function () {
              * @param {Function} calback callback function
              */
             pageview: function (payload, callback) {
+                var args = {},
+                    argsKeys = ['location', 'page', 'title'];
+
+                argsKeys.forEach(function (prop) {
```

Comment:
function name can be ```function consumer {xxx}``` if have
But why we need to debugging this anonymous function?